changes/987.feature.md

```diff
@@ -0,0 +1 @@
+Support fish shell
```

Comment:
Please write news fragment more concisely so that viewers can understand that this patch is to support `install-dev.sh` script working on fish shell.